### PR TITLE
Allow Solr commands to be defined via .lando.yml

### PR DESCRIPTION
**Pull request by CGoodwin90**
Allows Solr commands to be defined via .lando.yml & updates docker depenencies.

### CODE REVIEW

File: services/lagoon-solr/builder.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ module.exports = {
       if (_.get(options, '_app._config.uid', '1000') !== '1001') options._app.nonRoot.push(options.name);
 
       const solr = {
-        command: options.command,
+        command: `/sbin/tini -- /lagoon/entrypoints.sh ${options.command}`,
```
Comment (tobybellwood) [inline services/lagoon-solr/builder.js:30]: Does the default command in line 12 not need to then strip the entrypoints call to just be `solr-foreground`

File: .github/workflows/pr-all-services.yml [hunk 2]
```diff
@@ -34,7 +34,7 @@ jobs:
       # This block should eventually become use lando/actions-hyperdrive@v2
       - name: Verify Docker dependencies
         run: |
-          docker --version | grep "20.10."
+          docker --version | grep "23.0."
```
Comment (tobybellwood) [inline .github/workflows/pr-all-services.yml:37]: we'll need to find a better way to do this eventually - as the actions image updates, this check fails often (currently on 24.0.x)

File: docs/config.md [hunk 3]
```diff
@@ -98,11 +98,20 @@ The services we currently support with links to their associated Lagoon docs is
 * [PostgreSQL](https://docs.lagoon.sh/lagoon/docker-images/postgres)
 * [Redis](https://docs.lagoon.sh/lagoon/docker-images/redis)
 * [Ruby](https://docs.lagoon.sh/docker-images/ruby/)
-* [Solr](https://docs.lagoon.sh/lagoon/docker-images/solr)
+* [Solr*](https://docs.lagoon.sh/lagoon/docker-images/solr)
 * [Varnish](https://docs.lagoon.sh/lagoon/docker-images/varnish)
 
 Note that we are testing against the "Drupal" variants of the above but it's _possible_ the base services work as well.
 
+*Please note: if using [uselagoon/solr-8-drupal](https://github.com/uselagoon/lagoon-images/blob/main/images/solr-drupal/8.Dockerfile) you will need to add the following command to your `Solr` service in your `.lando.yml` file.
```
Comment (tobybellwood) [inline docs/config.md:106]: also mention that other solr images are supported here, and they may have different CMDs, this is just one example

**Review by tobybellwood — COMMENTED** [review]
We should also include a solr service (maybe not the drupal one - but a simple solr-8 with the correct CMD in the .lando.yml - and a test to check the mycore is there?

https://github.com/uselagoon/lagoon-images/blob/main/images/solr/8.Dockerfile#L78

**Comment by tobybellwood** [comment]
This looks good to me - @pirog or @reynoldsalec - ok for you?

File: .github/workflows/pr-all-services.yml [hunk 4]
```diff
@@ -32,10 +32,6 @@ jobs:
         run: yarn install --prefer-offline --frozen-lockfile
 
       # This block should eventually become use lando/actions-hyperdrive@v2
-      - name: Verify Docker dependencies
```
Comment (reynoldsalec) [inline .github/workflows/pr-all-services.yml:35]: Would be curious why this needs to get removed.

Comment (tobybellwood) [reply .github/workflows/pr-all-services.yml:35]: Github upgrades the version of Docker in their runners more frequently these days, but not all runners update at the same time. At the time of testing some were 23 and some 24 so failing tests.

File: docs/config.md [hunk 5]
```diff
@@ -98,11 +98,20 @@ The services we currently support with links to their associated Lagoon docs is
 * [PostgreSQL](https://docs.lagoon.sh/lagoon/docker-images/postgres)
 * [Redis](https://docs.lagoon.sh/lagoon/docker-images/redis)
 * [Ruby](https://docs.lagoon.sh/docker-images/ruby/)
-* [Solr](https://docs.lagoon.sh/lagoon/docker-images/solr)
+* [Solr*](https://docs.lagoon.sh/lagoon/docker-images/solr)
 * [Varnish](https://docs.lagoon.sh/lagoon/docker-images/varnish)
 
 Note that we are testing against the "Drupal" variants of the above but it's _possible_ the base services work as well.
 
+*Please note: if using [uselagoon/solr-8-drupal](https://github.com/uselagoon/lagoon-images/blob/main/images/solr-drupal/8.Dockerfile) you will need to add the following command to your `Solr` service in your `.lando.yml` file. Alternate Solr images are supported and may require different commands, this is just one example.
```
[the remark this answers is not in the document]
Comment (tobybellwood) [reply docs/config.md:106]: Mostly just this one, but if customers create their own cores, etc it can differ further.

**Pull request merged**